revolv/administrator/urls.py

```diff
@@ -10,4 +13,7 @@
     url(r'^$', is_administrator(AdministratorDashboardView.as_view()), name='dashboard'),
     url(r'^email$', is_administrator(AdministratorEmailView.as_view()), name='email'),
     url(r'^email/csv$', admin_email_csv_download, name='emailcsv'),
+    url(r'^accounting$', is_administrator(AdministratorAccountingView.as_view()), name='accounting'),
```

Comment:
nah, there's a separate url in the donor dashboard for donor access that doesn't check is_administrator